### PR TITLE
Docs: PyPi Setup & Deploy #68

### PR DESCRIPTION
**Pull request by yasheela-alla**
### Related Issue

Closes: #68 

### Description

This Documentation provides a comprehensive overview on how to set up and deploy a Python package to the Python Package Index (PyPI). 

### Checklist
- [x] I have followed the contribution guidelines
- [x] I have ensured my changes don't generate any new warnings or errors.
- [x] I have updated the documentation.
- [x] I have resolved all merge conflicts.

### CODE REVIEW

**Comment by yasheela-alla** [comment]
Hi @this-is-yaash @ShashwatAgrawal20 

Thank you for pointing that out! I’ve updated the changes requested.

If there are any further adjustments or suggestions you'd like, please let me know!

File: PyPi_Setup.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+│   ├── models.py                         # Module containing data models
+│   ├── piechart.py                       # Module for pie chart functionalities
+│   └── test.py                           # Test script for testing package modules
+│
+├── .gitignore                            # Specifies files to ignore in Git
+├── CHANGELOG.md                          # Log of changes made to the project
+├── CODE_OF_CONDUCT.md                    # Code of conduct for contributors
+├── LICENSE                               # License information for the project
+├── README.md                             # Documentation about the project
+└── setup.py                              # Setup script for packaging the project
```
Comment (this-is-yaash) [inline PyPi_Setup.md:67]: Just mention files that is related to setup & deploy.
`.github/`, `.gitignore`, `CHANGELOG.md`, `CODE_OF_CONDUCT.md` files are irrelevant

File: PyPi_Setup.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+│   └── pull_request_template.md          # Template for pull requests
+│
+├── package/                              # Main package directory
+│   ├── __init__.py                       # Required for package initialization
+│   ├── clp.py                            # Module for CLP functionality
+│   ├── display.py                        # Module for display-related functions
+│   ├── graph.py                          # Module for graph-related functions
+│   ├── models.py                         # Module containing data models
+│   ├── piechart.py                       # Module for pie chart functionalities
+│   └── test.py                           # Test script for testing package modules
```
Comment (this-is-yaash) [inline PyPi_Setup.md:60]: `stash` dir not mentioned!
It contains the files that is used for file integration!

Comment (this-is-yaash) [inline PyPi_Setup.md:60]: package dir has
1 forlder = `stash/` -> contains `test.py` for testing
4 files = `display`, `models`, `graph`, `piechart`

> please follow the actual structure

File: PyPi_Setup.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+├── CODE_OF_CONDUCT.md                    # Code of conduct for contributors
+├── LICENSE                               # License information for the project
+├── README.md                             # Documentation about the project
+└── setup.py                              # Setup script for packaging the project
+
+```
+### Note
+Feel free to change any filenames or directory names to better suit your needs. 
+
+The filenames in the stash/ directory are aliases, you can modify them as necessary!
```
Comment (this-is-yaash) [inline PyPi_Setup.md:73]: false!
do research!

File: PyPi_Setup.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+- **.github/**: Contains templates for issues and pull requests to streamline contributions.
+- **.gitignore**: Lists files and directories that should not be tracked by Git.
+- **changelog.md**: Documents significant changes and updates made to the project over time.
+- **code_of_conduct.md**: Outlines expected behavior and guidelines for contributors.
+- **setup.py**: Contains package information and dependencies.
+- **license**: Specifies the licensing terms under which the project is distributed.
+- **readme.md**: Provides an overview of the project, installation instructions, usage
+guidelines, and other relevant information.
+- **setup.py**: Contains metadata about the package and instructions on how to install it.
```
Comment (this-is-yaash) [inline PyPi_Setup.md:85]: explain relevant files

File: PyPi_Setup.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+1. **Upload the Package Using Twine**:
+
+    - First, ensure you have Twine installed. If not, you can install it using pip:
+     ```bash
+     pip install twine
+     ```
+   - Next, navigate to the directory containing your package files in the terminal.
+   - Create a source distribution of your package by running:
+     ```bash
+     python setup.py sdist
```
Comment (this-is-yaash) [inline PyPi_Setup.md:156]:
```suggestion
     python setup.py sdist bdist_wheel
```

**Review by this-is-yaash — COMMENTED** [review]
It has lot of mistakes, I can spot!

Please conduct proper research and work on the documentation!

> Avoid using GPTs, since you can see the DOC is not useful!

Try to do it on your own, get understanding of what the project does!


I would say revamp the whole doc, with your own content!
Keep it simple, short & clear!

**Comment by Wahid7852** [comment]
Any updates @yasheela-alla

**Pull request closed without merging**